### PR TITLE
Add Authentication and Registration endpoints. Add dummy data to User…

### PR DESCRIPTION
**Pull request by JadiellMV**
Added Authentication and Registration endpoints, and also dummy data to the users table to be able to work with some data. This pull request was done collaboratively between @AnthonyMed13 and @JadiellMV, and takes care of task #24, task #18, and task #17.

### CODE REVIEW

**Review by AngelCIICMorales — CHANGES_REQUESTED** [review]
There is conflict. Please make sure the branch is up to date.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (FrancesC8) [inline package.json:1]: The changes for this package.json should be in /backend/package.json.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,1578 @@
+{
```
Comment (FrancesC8) [inline package-lock.json:1]: This file should not exist here. There should only be package-lock.json files in the backend and frontend folders

**Comment by AnthonyMed13** [comment]
> There is conflict. Please make sure the branch is up to date.

Everything is up to date now

**Review by AngelCIICMorales — APPROVED** [review]
LGTM!

**Review by AngelCIICMorales — CHANGES_REQUESTED** [review]
Please fix the merge conflicts.

**Review by sandy-marrero — APPROVED** [review]

**Comment by AnthonyMed13** [comment]
> Please fix the merge conflicts.

Done.

**Review by AngelCIICMorales — APPROVED** [review]
LGTM.

**Pull request merged**